api/src/guards/jwt-auth.guard.ts

```diff
@@ -29,10 +35,28 @@ export class JwtAuthGuard extends AuthGuard('jwt') {
     return super.canActivate(context);
   }
 
-  handleRequest(err, user) {
+  handleRequest(err: any, user: any, info: any, ctx: ExecutionContext): any {
     if (err || !user) {
       throw err || new UnauthorizedException();
     }
-    return user;
+
+    const authorizationChecks = this.reflector.get<AuthorizationCheck[]>(
+      AUTHORIZATION_CHECKS,
+      ctx.getHandler(),
+    );
+    if (authorizationChecks === undefined) return user;
+
+    for (let idx = 0; idx < authorizationChecks.length; idx++) {
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
the fact that we need to disable linting rules should set an alarm on our minds that something we are doing is not quite right